src/pages/landing/landing.css

```
@@ -0,0 +1,749 @@
+/* ============================================================
+   WhyLog Landing page — scoped styles
+   Ported from the design bundle (WhyLog Landing.html).
+   ============================================================ */
+
+.whylog-landing {
+  font-family: "Pretendard", "Pretendard Variable", "Inter", sans-serif;
+  color: #1c1c28;
+  background: #fafafc;
+  font-feature-settings: "ss01", "tnum", "cv02", "cv03", "cv04";
+  -webkit-font-smoothing: antialiased;
+  text-rendering: optimizeLegibility;
```

Stylelint (17.11.0)

[error] Expected "optimizeLegibility" to be "optimizelegibility" (value-keyword-case)

(value-keyword-case)